.changeset/cool-rockets-confess.md

```diff
@@ -0,0 +1,11 @@
+---
+'@spreadshirt/backstage-plugin-s3-viewer-backend': patch
+---
+
+Replace some deprecations caused by the removal of the backend-tasks plugin and deprecate old backend system.
+Please, remove any usage of `RouterOptions` and `createRouter`.
```

Comment:
Let's link https://github.com/spreadshirt/backstage-plugin-s3/tree/main/plugins/s3-viewer-backend here so people know where to go.

I think that README also should be adjusted to only mention the new backend.

Reply:
Added a note at the end. And also added a deprecation warning in the readme. The whole block will be removed together with the deprecated types. I think it's also good to do it this way if you like :)